libs/core/langchain_core/output_parsers/json.py

```diff
@@ -13,9 +13,9 @@
 from langchain_core.output_parsers.transform import BaseCumulativeTransformOutputParser
 from langchain_core.outputs import Generation
 from langchain_core.utils.json import (
-    parse_and_check_json_markdown,
+    parse_and_check_json_markdown,  # noqa: F401 backward compatibility
```

Comment:
hmm i don't think this is the correct way to specify that this is for backwards compatibility